spec/models/authorization_document_spec.rb

```diff
@@ -8,5 +8,12 @@
 
   describe "validations" do
     it { is_expected.to validate_presence_of(:expires_at) }
+
+    describe "enumerators" do
+      it "should define an enum for description" do
```

Comment:
Evita usar `should` na descrição dos seus testes.
Use ` it "defines an enum ...`